rdtools/plotting.py

```diff
@@ -395,18 +395,19 @@ def availability_summary_plots(power_system, power_subsystem, loss_total,
     ax3 = fig.add_subplot(gs[2, 0], sharex=ax1)
     ax4 = fig.add_subplot(gs[:, 1], sharex=ax1)
 
-    # inverter power
-    power_system.plot(ax=ax1)
-    ax1.set_ylabel('Inverter Power [kW]')
     # meter power
```

Comment:
At the risk of nitpicking, should this comment be "inverter power"?

Reply:
no, but the one a few lines down should be.  d'oh